packages/web-components/src/components/footer/__stories__/footer.stories.ts

```diff
@@ -13,24 +13,33 @@ import inPercy from '@percy-io/in-percy';
 import { FOOTER_SIZE } from '../footer';
 import '../footer-composite';
 import '../footer-container';
+import mockLangList from './language-list';
 import mockLinks from './links';
 import mockLegalLinks from './legal-links';
 import mockLocaleList from '../../locale-modal/__stories__/locale-data.json';
 import readme from './README.stories.mdx';
+import styles from './footer.stories.scss';
 
 export const base = ({ parameters }) => {
-  const { langDisplay, language, size, legalLinks, links, localeList } = parameters?.props?.FooterComposite ?? {};
+  const { langDisplay, language, size, langList, legalLinks, links, localeList, enableLanguageSelector } =
+    parameters?.props?.FooterComposite ?? {};
   const { useMock } = parameters?.props?.Other ?? {};
+
   return html`
+    <style>
+      ${styles}
+    </style>
     ${useMock
       ? html`
           <dds-footer-composite
             language="${ifNonNull(language)}"
             lang-display="${ifNonNull(langDisplay)}"
             size="${ifNonNull(size)}"
+            languageList="${ifNonNull(langList)}"
             .legalLinks="${ifNonNull(legalLinks)}"
             .links="${ifNonNull(links)}"
             .localeList="${ifNonNull(localeList)}"
+            .enableLanguageSelector="${enableLanguageSelector}"
```

Comment:
Please also see `@property` definition of `enable-language-selector`:

```suggestion
            ?enable-language-Selector="${enableLanguageSelector}"
```